extendeddatasquare_test.go

```diff
@@ -1,37 +1,75 @@
 package rsmt2d
 
 import (
+	"bytes"
 	"crypto/rand"
 	"encoding/json"
 	"fmt"
 	"reflect"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	zero     = bytes.Repeat([]byte{0}, 64)
+	one      = bytes.Repeat([]byte{1}, 64)
+	two      = bytes.Repeat([]byte{2}, 64)
+	three    = bytes.Repeat([]byte{3}, 64)
+	four     = bytes.Repeat([]byte{4}, 64)
+	five     = bytes.Repeat([]byte{5}, 64)
+	eight    = bytes.Repeat([]byte{8}, 64)
+	eleven   = bytes.Repeat([]byte{11}, 64)
+	thirteen = bytes.Repeat([]byte{13}, 64)
+	fifteen  = bytes.Repeat([]byte{15}, 64)
 )
 
 func TestComputeExtendedDataSquare(t *testing.T) {
```

Comment:
unit tests in this file had to be updated because the existing tests use shard sizes of size 1 which will result in an error from klauspost. E.g.

```
--- FAIL: TestMarshalJSON (0.00s)
panic: shard sizes do not match [recovered]
	panic: shard sizes do not match
```

Reply:
ahh sorry just saw this comment, mind adding a comment in the code to have the same reminder for future readers?

Reply:
Good idea! Resolved by https://github.com/celestiaorg/rsmt2d/pull/154/commits/fb77f6699d289506c09e98e03a95a4bee064af58